ui-templates/custom-types/unit_data.json

```diff
@@ -0,0 +1,21 @@
+{
+    "unit_data": {
+        "python::module": "simulariumio",
+        "python::object": "UnitData",
+        "parameters": {
+            "name": {
+                "name": "Name",
+                "data_type": "text",
+                "description": "Provide the unit name (we support this list https://github.com/hgrecco/pint/blob/master/pint/default_en.txt)",
+                "required": true
+            },
+            "magnitude": {
+                "name": "Magnitude",
+                "data_type": "number",
+                "description": "Provide a multiplier in case values are not in whole units",
+                "required": false,
```

Comment:
so it's ok to make `UnitData({name: "nm"})`? I feel like i've seen it `UnitData("nm")`

Reply:
I *think* it errored out when I tried the first thing, but something else may have gone wrong. do you have a test that does it both ways?